resources/schema.graphql

```diff
@@ -265,6 +263,27 @@ type CompactionMetadataOnly {
   recursive: Boolean!
 }
 
+union CompareChainsResult = CompareChainsResultStatus | CompareChainsResultError
+
+type CompareChainsResultError {
+  reason: CompareChainsResultReason!
+}
+
+type CompareChainsResultReason {
+  message: String!
+}
+
+type CompareChainsResultStatus {
+  message: CompareChainsStatus!
+}
+
+enum CompareChainsStatus {
+  EQUAL
+  BEHIND
+  AHEAD
+  DIVERGED
+}
+
```

Comment:
Double check schema merging - this content is not from the private datasets branch

Reply:
Sync with schema from master